.goreleaser.yml

```diff
@@ -16,13 +16,14 @@ builds:
   - amd64
   ldflags: []
 
-archive:
-  name_template: "{{.ProjectName}}-{{.Os}}-{{.Arch}}"
-  format_overrides:
-  - goos: windows
-    format: zip
-  files:
-    - none*  # only package the binary - not defaults: readme, license, changelog
+archives:
+  -
```

Comment:
don't leave this on a line by itself (it's hard to read) and probably needs an `id`. https://github.com/cyberark/secretless-broker/blob/master/.goreleaser.yml#L79-L80.